docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/DockerJDBCIntegrationSuite.scala

```diff
@@ -112,7 +112,7 @@ abstract class DockerJDBCIntegrationSuite
       // Start the container and wait until the database can accept JDBC connections:
       docker.startContainer(containerId)
       jdbcUrl = db.getJdbcUrl(dockerIp, externalPort)
-      eventually(timeout(60.seconds), interval(1.seconds)) {
+      eventually(timeout(200.seconds), interval(200.seconds)) {
```

Comment:
@yhuai  This line of timeout is required for the oracle related test.Because, the time taken by the oracle instance to bring up would not be enough with the old setting. We tested it in our local and  
confirmed this. If we do not change this time duration, Oracle would throw an exception, saying that the instance was tried to hit during the initialization stage.